### ISSUE TITLE
Support the new WCAG 2.2 guidelines

### ISSUE THREAD

**Issue by mgifford** [GSA/openacr#352]
OpenACR should be keeping up with WCAG.

**Comment by mgifford** [comment]
We've got an initial PR up here CivicActions#21 which we hope to migrate in at some point.

**Comment by patrickhlauke** [comment]
Made a naive attempt on my part (at least for the catalog stuff) https://github.com/GSA/openacr/pull/353

**Comment by mgifford** [comment]
Great to see it incorporated in. Thanks for your contribution @patrickhlauke 

https://github.com/GSA/openacr/pull/353
